src/main/java/frc/robot/swerve/SwerveConfig.java

```diff
@@ -4,7 +4,6 @@
 import com.ctre.phoenix.motorcontrol.InvertType;
 import com.ctre.phoenix.motorcontrol.NeutralMode;
 import com.ctre.phoenix.motorcontrol.can.TalonFX;
-import com.revrobotics.CANSparkMax.SoftLimitDirection;
```

Comment:
If you import a class, but don't use it, it's "unused" and Visual Studio will highlight it with a squiggly yellow line. It won't necessarily break anything, but it's helpful to keep the code clean by removing them.